src/graphx/graphx.asm

```diff
@@ -460,7 +459,6 @@ _GetPixel:
 	inc	hl
 	inc	hl
 	inc	hl                          ; move to next argument
-	ld	de,0
```

Comment:
This removal is not safe. Although _PixelPtr_ASM is documented as the y input being only 8-bit, the full 24-bit value is used for clipping.